File: src/main/java/org/ankol/server/services/ClashDataOperationService.java
package org.ankol.server.services;

public interface ClashDataOperationService {
    /**
     * 触发更新突袭周末数据的操作
     *
     * @return {@link Boolean} 成功或失败
     */
    boolean triggerRaidSeasonUpdate();

    /**
     * 更新部落成员信息
     */
    void triggerClanMemberUpdate();
}

File: src/main/java/org/ankol/server/services/ClashDataOperationService.kt
package org.ankol.server.services

import org.ankol.server.dao.entity.ClanMemberEntity

interface ClashDataOperationService {
    /**
     * 触发更新突袭周末数据的操作
     *
     * @return [Boolean] 成功或失败
     */
    fun triggerRaidSeasonUpdate(): Boolean

    /**
     * 更新部落成员信息
     * @return 最新的成员信息集合
     */
    fun triggerClanMemberUpdate(): List<ClanMemberEntity>
}
